tests/generators/test_nim.py

```diff
@@ -45,6 +46,20 @@ def test_nim_parallel_attempts():
     assert True
 
 
+@pytest.mark.skipif(
+    os.getenv(NVOpenAIChat.ENV_VAR, None) is None,
+    reason=f"NIM API key is not set in {NVOpenAIChat.ENV_VAR}",
+)
+def test_nim_parallel_requests():
+    _config.system.parallel_requests = 2
+
+    g = NVOpenAIChat(name="nvidia/nemotron-4-340b-instruct")
```

Comment:
Can we use a local, test generator, so this test always runs successfully? test.Lipsum is fine.

Also, `_plugins.load_plugin()` is often a stabler/easier route for instantiating generators

Reply:
Updated